developer_guide.md

```diff
@@ -0,0 +1,94 @@
+# Development guidelines for Router Sidecar
+
+The Slice VPN Gateway is a slice network service component that provides a secure VPN tunnel between any two clusters that are a part of the slice.
+
+## Building and Installing `gateway-sidecar` in a Local Kind Cluster
+For more information, see [getting started with kind clusters](https://docs.avesha.io/opensource/getting-started-with-kind-clusters).
```

Comment:
Replace the doc link with https://docs.avesha.io/documentation/open-source/0.2.0/getting-started-with-kind-clusters